src/components/Login/index.stories.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Login from "./index";
+
+storiesOf("Login", module).add("default", () => <Login />);
```

Comment:
Right now we can't storybook all the variations of the Login component because all the state and the actual components are meshed inside. The route level should still have a lot of the state management and network request functions, but the login form is still coupled and needs to be grouped in another component to show all variations just based on props alone